Formula/qemu.rb

```diff
@@ -140,6 +65,7 @@ def install
       --enable-virtfs
       --extra-cflags=-DNCURSES_WIDECHAR=1
       --disable-sdl
+      --disable-linux-user
```

Comment:
Could you add a comment line to explain that this should be enabled again when the CI gets updated to a recent version of Ubuntu?